File: backend/app/modules/running_bills/permissions.py
from app.modules.identity.enums import PermissionKey

RUNNING_BILL_PERMISSIONS = {
  PermissionKey.RUNNING_BILL_READ: "View running bills / IPCs for projects.",
  PermissionKey.RUNNING_BILL_CREATE: "Generate running bills from approved progress claims.",
  PermissionKey.RUNNING_BILL_ISSUE: "Issue or cancel running bills.",
}

File: frontend/src/modules/running_bills/permissions.ts
export const RUNNING_BILL_PERMISSIONS = {
  RUNNING_BILL_READ: "running_bill:read",
  RUNNING_BILL_CREATE: "running_bill:create",
  RUNNING_BILL_ISSUE: "running_bill:issue",
} as const;

export type RunningBillPermission =
  (typeof RUNNING_BILL_PERMISSIONS)[keyof typeof RUNNING_BILL_PERMISSIONS];